src/app/public/modules/inline-delete/inline-delete.component.scss

```diff
@@ -0,0 +1,39 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+
+.sky-inline-delete {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, .5);
+  z-index: 1;
```

Comment:
What happens if the inline delete is in front of something else with a z-index, like a dropdown?

Reply:
Changed this to 999 which is just below the omnibar